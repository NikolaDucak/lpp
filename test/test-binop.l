// bin ops
println(
	// arithmetic
	10+10,
	13-3.5,
	150-200,

	10/10,
	5/3,
	130*1.5,
	
	// comparison,
	5  >  5,
	10 <  20,
	5  >= 5,
	10 <  0,
	5.1<= 5.1

	// logic ops
	true  and true,
	true  and false,
	false and true,
	false and false,

	true  or true,
	true  or false,
	false or true,
	false or false,
);
