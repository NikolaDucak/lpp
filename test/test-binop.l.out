20
9.5
-50
1
1.666667
195
false
true
true
false
true
true
false
false
false
true
true
true
false
